analysis_options.yaml

```diff
@@ -21,7 +21,7 @@ analyzer:
   ]
   strong-mode:
     implicit-casts: false
-    implicit-dynamic: false
+    # implicit-dynamic: false # at least temporary for RecursiveVisitor change
```

Comment:
I'd like to avoid doing this - can we just opt-in the line specifically?

Reply:
TODO added. I can't make opting out only the line work :(